samples/highcharts/demo/advanced-accessible/demo.js

```diff
@@ -77,100 +79,59 @@ var chart = Highcharts.chart('container', {
                     point.median + ', high ' + point.high;
             }
         },
-
         series: {
             descriptionFormat: '{series.name}, series {seriesNumber} of {chart.series.length} with {series.points.length} data points.'
         },
-
         typeDescription: 'Low, median, high. Each data point has a low, median and high value, depicted vertically as small ticks.' // Describe the chart type to screen reader users, since this is not a traditional boxplot chart
     },
-
+    legend: {
+        enabled: false
+    },
     xAxis: [{
         accessibility: {
-            description: 'Months of the year'
+            description: 'Countries'
         },
-        categories: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+        categories: ['Finland', 'Iceland', 'Norway', 'Sweden', 'Japan', 'Russia', 'France', 'UK', 'Canada'],
         crosshair: true
     }],
-
     yAxis: {
         title: {
-            text: 'Fruits consumed'
+            text: 'Age'
         },
         min: 0
     },
-
     responsive: {
         rules: [{
             condition: {
                 maxWidth: 550
             },
             chartOptions: {
                 xAxis: {
-                    categories: ['Jan', 'Feb', 'Mar', 'Apr', 'May', 'Jun', 'Jul', 'Aug', 'Sep', 'Oct', 'Nov', 'Dec']
+                    categories: ['Finland', 'Iceland', 'Norway', 'Sweden', 'Japan', 'Russia', 'France', 'UK', 'Canada']
                 }
             }
         }]
     },
-
     tooltip: {
         shared: true
     },
-
     plotOptions: {
         series: {
             stickyTracking: true,
             whiskerWidth: 5
         }
     },
-
     series: [{
```

Comment:
It's not overly critical that it has multiple series, but it does need to have `null` points. It is supposed to be a more complex demo, so it would be good if it had at least 2 series.

Reply:
have changed back to the old data for the time being. will look for more relevant data.